react/features/calendar-sync/components/ConferenceNotification.native.js

```diff
@@ -0,0 +1,242 @@
+// @flow
+import React, { Component } from 'react';
+import {
+    Dimensions,
+    Text,
+    TouchableOpacity,
+    View
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import { appNavigate } from '../../app';
+import { getURLWithoutParams } from '../../base/connection';
+import { getLocalizedDateFormatter, translate } from '../../base/i18n';
+import { Icon } from '../../base/font-icons';
+
+import styles from './styles';
+
+const ALERT_MILLISECONDS = 5 * 60 * 1000;
+
+type Props = {
+
+    /**
+     * The Redux dispatch function.
+     */
+    dispatch: Function,
+
+    /**
+     * The URL of the current conference without params.
+     */
+    _currentConferenceURL: string,
+
+    /**
+     * The calendar event list.
+     */
+    _eventList: Array<Object>,
+
+    /**
+     * The height of the screen. Needed to be in props to trigger a re-render.
+     */
+    _screenHeight: number,
+
+    /**
+     * The translate function.
+     */
+    t: Function
+};
+
+type State = {
+
+    /**
+     * The event object to display the notification for.
+     */
+    event?: Object
+};
+
+/**
+ * Component to display a permanent badge-like notification on the
+ * conference screen when another meeting is about to start.
```

Comment:
Is there a way to dismiss the notification? Does it matter for now?

Reply:
There's no way now, because then it's not different from a normal system notification. Do you think there should be a way?

Reply:
@virtuacoplenny I created a ticket for this, I think this would be a good addition